invisible_cities/conftest.py

```diff
@@ -699,6 +704,44 @@ def deconvolution_config(ICDIR, ICDATADIR, PSFDIR, config_tmpdir):
 
     return conf, PATH_OUT
 
+<<<<<<< HEAD
+=======
+
+@pytest.fixture(scope='function') # Needs to be function as the config dict is modified when running
```

Comment:
Is the comment outdated? I've seen that you copy the config in each test, no?

Reply:
I think the comment is fine. This only applies to the city, the copied ones are for the `trigger_functions`.